.github/workflows/release.yml

```diff
@@ -237,6 +237,35 @@ jobs:
           ./target/release/linkerd2-cli-*-linux.sha256
           ./target/release/linkerd2-cli-*-windows.exe
           ./target/release/linkerd2-cli-*-windows.exe.sha256
+  website_publish_check:
+    name: Linkerd website publish check
+    if: startsWith(github.ref, 'refs/tags/stable') || startsWith(github.ref, 'refs/tags/edge')
```

Comment:
(un)related to this PR, can we assume that this entire workflow is for edge/stable releases, and then avoid these `startsWith` lines, instead filtering at the top, something like:
```yaml
name: Release
on:
  push:
    tags:
    - "edge-*"
    - "stable-*"
```

Reply:
@siggy I was thinking about this and I feel like it's not something we need to change right now. There may some situation where we'd like to publish a tag that is not edge or release related--maybe it's just for testing something.

It would still be helpful to have this workflow run so that the images are pushed to the linkerd registry after being fully tested, but not attempt to create a new GH release and update the website.

That's all a hypothetical though, so maybe it makes sense!